span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -159,7 +167,7 @@ public RawSpan convert(Span jaegerSpan) throws Exception {
 
       // build raw span
       RawSpan rawSpan = rawSpanBuilder.build();
-      LOG.debug("Converted Jaeger span: {} to rawSpan: {} ", jaegerSpan, rawSpan);
+      logSpanConversion(jaegerSpan, rawSpan);
```

Comment:
call this method under `LOG.isDebugEnabled()`?

Reply:
Yeah. We can avoid the method call fully in regular code path, as opposed to returning early from that method.
Do you know if JVM would already do such optimization? I've to refresh my memory on that :)